client/components/EyesWanted/AddEyesWanted.vue

```diff
@@ -77,7 +107,8 @@
         } catch (e) {
           this.$set(this.alerts, e, 'error');
           setTimeout(() => this.$delete(this.alerts, e), 3000);
-        }
+        };
+        this.EyesWantedRequest('');
```

Comment:
replace calls to this.EyesWantedRequest('') to this.$store.commit('refreshAllEyesWanted'); or whatever appropriate call it is now